### ISSUE TITLE
make latexpdf` failed with ├ (U+251C) character

### ISSUE THREAD

**Issue by LuisBL** [sphinx-doc/sphinx#3851]
Subject: `make latexpdf` failed with ├ (U+251C) character

### Problem
- pdflatex phase failed if there is a ├ (U+251C) character

with an unique `doc.rst`::

```
$ more  doc.rst
Title
=======
Illustrate a tree with unix `tree` cmd::

  $ tree
  .
  └── f11
      ├── f111
      ├── f112
      │   └── README
      └── f113
  $
```

#### Procedure to reproduce the problem
```
$ make latexpdf`
```

#### Error logs / results
```
/usr/share/texlive/texmf-dist/tex/latex/psnfss/t1pcr.fd)
<use corp_logo_200x65px2.png>
(/usr/share/texlive/texmf-dist/tex/latex/psnfss/ts1ptm.fd) [1 <./corp_logo_
200x65px2.png>]

! Package inputenc Error: Unicode char ├ (U+251C)
(inputenc)                not set up for use with LaTeX.

See the inputenc package documentation for explanation.
Type  H <return>  for immediate help.
 ...                                                                                    
l.176 ...sa \PYGZhy{}\PYGZgt{} 
```
#### Expected results
Get the result even with ├ (U+251C) cararcter in source

### Environment info
- OS: Linux Ubuntu 16.10
- Python version: 3.5.2
- Sphinx version: Sphinx v1.5.5

**Comment by jfbu** [comment]
@LuisBL please try again after adding this to `latex_elements` in conf.py

    'preamble': r'\usepackage{pmboxdraw}',

There is no TeX font usable by pdflatex which provides the characters (to the best of my knowledge), but package `pmboxdraw` emulates them with horizontal and vertical rules. The difference with font glyphs is seen when copying pasting from PDF, you don't get them.

If you switch to `xelatex` or `lualatex` then it is only matter to use a font with such characters available.

However, there is issue #3706 which affects also `pdflatex + \usepackage{pmboxdraw}` 

For details see last paragraph of https://github.com/sphinx-doc/sphinx/issues/3289#issuecomment-299431200

@tk0miya I am not sure this is bug. It is problem of `pdflatex` not problem of Sphinx. I don't know better solution for `pdflatex` than using package `pmboxdraw`.

Relates: #3706

**Comment by jfbu** [comment]
You can additionally to usage of `pmboxdraw` incorporate this at bottom of your conf.py:

```
def setup(app):
    from sphinx.util.texescape import tex_replacements
    tex_replacements.remove(('─', r'-'))
    tex_replacements.remove(('│', r'\textbar{}'))
```

to circumvent issue #3706. Result:

![capture d ecran 2017-06-04 a 10 09 51](https://cloud.githubusercontent.com/assets/2589111/26759977/fb8d67ac-490d-11e7-9ff1-41d6e8bb92e3.png)

nota bene: `pmbowdraw` has no provisions for diagonal characters. Thus, if you need ``╲``, then you may use `xelatex` or `lualatex` and this also

        tex_replacements.remove(('╲', r'\textbackslash{}'))

together with use of suitable font, for example

```
    'preamble': r'''
\setmainfont{FreeSerif}
\setsansfont{FreeSans}
\setmonofont{FreeMono}
''',
```
However there is then a problem of slight vertical gaps between lines. (the characters occupy less vertical space than the LaTeX assigned baseline size) 

But the `xelatex/lualatex` result is copyable-pastable from PDF including the BOX DRAWINGS characters. With `pdflatex` you don't have the problem of vertical gaps but the characters are emulated by rules in the PDF, hence don't get copied on text selection in PDF. Also `pmboxdraw` handles only characters which can draw with vertical and horizontal rules.

**Comment by tk0miya** [comment]
Fixed by #3876
